OSHI doesn't build on JDK 10

Command: `mvn package`

```
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running oshi.SystemInfoTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.875 s - in oshi.SystemInfoTest
[INFO] Running oshi.util.EdidUtilTest
[ERROR] Tests run: 4, Failures: 0, Errors: 4, Skipped: 0, Time elapsed: 0.013 s <<< FAILURE! - in oshi.util.EdidUtilTest
[ERROR] testGetEdidAttrs(oshi.util.EdidUtilTest)  Time elapsed: 0.007 s  <<< ERROR!
java.lang.NoClassDefFoundError: javax/xml/bind/DatatypeConverter
	at oshi.util.EdidUtilTest.<clinit>(EdidUtilTest.java:57)
Caused by: java.lang.ClassNotFoundException: javax.xml.bind.DatatypeConverter
	at oshi.util.EdidUtilTest.<clinit>(EdidUtilTest.java:57)
```
I think an --add-modules VM parameter is needed. See https://github.com/checkstyle/checkstyle/issues/5027

--- a/oshi-core/src/test/java/oshi/util/EdidUtilTest.java
+++ b/oshi-core/src/test/java/oshi/util/EdidUtilTest.java
@@ -21,8 +21,6 @@ package oshi.util;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
 
-import javax.xml.bind.DatatypeConverter;
-
 import org.junit.Test;
 
 /*
@@ -54,7 +52,7 @@ public class EdidUtilTest {
     private final static String EDID_STR2 = EDID_HEADER + EDID_MANUFID + EDID_PRODCODE + EDID_SERIAL + EDID_WKYR
             + EDID_VERSION + EDID_VIDEO + EDID_TIMING + EDID_DESC5 + EDID_DESC6 + EDID_DESC7 + EDID_DESC8 + EDID_EXTS
             + EDID_CHKSUM;
-    private final static byte[] EDID = DatatypeConverter.parseHexBinary(EDID_STR);
+    private final static byte[] EDID = ParseUtil.hexStringToByteArray(EDID_STR);
 
     @Test
     public void testToHexString() {
@@ -112,7 +110,7 @@ public class EdidUtilTest {
     public void testToString() {
         String[] toString = EdidUtil.toString(EDID).split("\\n");
         assertEquals(6, toString.length);
-        toString = EdidUtil.toString(DatatypeConverter.parseHexBinary(EDID_STR2)).split("\\n");
+        toString = EdidUtil.toString(ParseUtil.hexStringToByteArray(EDID_STR2)).split("\\n");
         assertEquals(6, toString.length);
     };
 }

